--- a/firebase.js
+++ b/firebase.js
@@ -1,11 +1,22 @@
-import { initializeApp } from "firebase/app";
-
-const firebaseConfig = {
-  apiKey: "AIzaSyDJhjHc9xsxfD8jMRtu0mFmamScGrJzCbs",
-  authDomain: "fir-eed0e.firebaseapp.com",
-  projectId: "fir-eed0e",
-  storageBucket: "fir-eed0e.appspot.com",
-  messagingSenderId: "702932216448",
-  appId: "1:702932216448:web:d73b9538544b2ff3ca1465",
-  measurementId: "G-BE7T69C8Q9",
-  databaseURL: "https://fir-eed0e-default-rtdb.firebaseio.com"
-};
-
-// Initialize Firebase
-export const app = initializeApp(firebaseConfig);
+import { initializeApp } from "firebase/app";
+
+const firebaseConfig = {
+  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
+  authDomain: process.env.REACT_APP_FIREBASE_AUTH_DOMAIN,
+  projectId: process.env.REACT_APP_FIREBASE_PROJECT_ID,
+  storageBucket: process.env.REACT_APP_FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGING_SENDER_ID,
+  appId: process.env.REACT_APP_FIREBASE_APP_ID,
+  measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENT_ID,
+  databaseURL: process.env.REACT_APP_FIREBASE_DATABASE_URL,
+};
+
+// Initialize Firebase
+export const app = initializeApp(firebaseConfig);